flink-autoscaler/src/main/java/org/apache/flink/autoscaler/config/AutoScalerOptions.java

```diff
@@ -219,6 +219,22 @@ private static ConfigOptions.OptionBuilder autoScalerConfig(String key) {
                     .withDescription(
                             "Processing rate increase threshold for detecting ineffective scaling threshold. 0.1 means if we do not accomplish at least 10% of the desired capacity increase with scaling, the action is marked ineffective.");
 
+    public static final ConfigOption<Double> GC_PRESSURE_THRESHOLD =
+            autoScalerConfig("memory.gc-pressure.threshold")
+                    .doubleType()
+                    .defaultValue(0.3)
+                    .withFallbackKeys(oldOperatorConfigKey("memory.gc-pressure.threshold"))
+                    .withDescription(
+                            "Max allowed GC pressure (percentage spent garbage collecting) during scaling operations. Autoscaling will be paused if the GC pressure exceeds this limit.");
+
+    public static final ConfigOption<Double> HEAP_USAGE_THRESHOLD =
+            autoScalerConfig("memory.heap-usage.threshold")
+                    .doubleType()
+                    .defaultValue(1.)
+                    .withFallbackKeys(oldOperatorConfigKey("memory.heap-usage.threshold"))
```

Comment:
`oldOperatorConfigKey` is not needed, right?

`GC_PRESSURE_THRESHOLD` is same as well.

Reply:
true, removing

Reply:
After a brief discussion with @mxm offline, we feel that it's easiest and harmless to keep the fallback keys consistent for all options. We may be able to remove these at a future point (a new major version of the operator) but it's simply better to have a consistent fallback syntax for operator users.

Reply:
I agree, we should support the old-style config namespace until the next major release. The overhead of this is very low and our users do not have to worry about using the right prefix for their configuration. As things stand, even if we removed the old namespace, we would have to add a mapping in the operator (either public our in our fork) because it is not feasible to go to every user to ask them to change their configs.

Reply:
Thanks for the clarification!

Let's respect this rule in the future~